### ISSUE TITLE
NVDA 2015.1 reports locale 'zh_HK' cannot be set at startup

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5017]
**Reported by TimothyLee on 2015-04-02 14:18**
When running NVDA 2015.1 under Hong Kong version of Windows 7, an error dialog is shown reporting:  locale 'zh_HK' cannot be set.

When clicking on the "Details" button, the following messages are shown:

  Cannot set locale to language "Chinese (Hongkong)".
  locale 'zh_HK' cannot be set.

This error does not appear to affect the actual operation of NVDA.

The error dialog can be triggered on any computer by choosing zh_HK as the language of NVDA through the "General Settings" dialog.

The problem appears to be caused by the transition to wxPython 3, which seems to have no support for the zh_HK locale.

**Comment by jcsteh** [comment]
I set my language to Chinese Hong Kong and can't reproduce this; NVDA started normally. Therefore, I believe this is fixed. Please reopen if not.
